cmake/QSkinnyTools.cmake

```diff
@@ -8,11 +8,18 @@
 function(qsk_svg2qvg SVG_FILENAME QVG_FILENAME)
     get_filename_component(QVG_FILENAME ${QVG_FILENAME} ABSOLUTE)
     get_filename_component(SVG_FILENAME ${SVG_FILENAME} ABSOLUTE)
+
+    if(TARGET Qt6::Svg)
+        set(QtSvgTarget Qt6::Svg)
+    elseif(TARGET Qt5::Svg)
+        set(QtSvgTarget Qt5::Svg)
+    endif()
```

Comment:
For some reason `$<TARGET_FILE_DIR:Qt::Svg>` doesn't work for `ALIAS`? Qt::Svg, but even if it would still break if the user configures `QT_NO_CREATE_VERSIONLESS_TARGETS` ... 😢